docs/configuration/settings.rst

```diff
@@ -0,0 +1,586 @@
+Settings
+########
+
+.. image:: /images/overview-features.png
+    :align: center
+
+Kinto is built to be highly configurable. As a result, the related
+configuration can be verbose, but don't worry, all configuration flags are
+listed below.
+
+.. note::
+
+    In order to ease deployment or testing strategies, *Kinto* reads settings
+    from environment variables, in addition to ``.ini`` files.
+
```

Comment:
I would also *explicitly state the order of operations*, which (based on the example below) would appear to be:
`ENV > .ini > internal_default`.